tests/loader/resources/loader_test_directory/sub_dir_a/test_d.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2015 Confluent Inc.
```

Comment:
```suggestion
# Copyright 2020 Confluent Inc.
```

Reply:
do we care about copyright on the unit-test resources? Maybe just remove it entirely?

Reply:
Its your call.